### DISCUSSION TITLE
如何修改标题和人名？

### DISCUSSION THREAD

**Discussion by tornadoPark** [sjtug/SJTUThesis#700] [提问] [answered]
win10，Tex live，不过这好像跟问题没啥关系
就是在哪里可以修改论文标题、人名、学号这些基本信息
还有摘要在\contents\abstract.tex中，那摘要下的关键词在什么地方指定呢？

完全不懂，求大佬带。。

**Comment by skyzh** [comment, marked as answer]
setup.tex
